.gitignore

```diff
@@ -26,6 +26,7 @@ etc/conf/*.der
 connectors/spark/etc/conf/certs.json
 connectors/spark/etc/conf/*.txt
 connectors/spark/etc/conf/*.der
+.java-version
```

Comment:
I actually want to keep this, this is the file created by jenv when using multiple java versions

Reply:
Can you ensure the JAVA_HOME env variable in your terminal is configured to point to JDK17 and if you still need this?

Reply:
setting the JAVA_HOME env variable does work. However jenv is helping here by allowing me to have project level java versions based on this file i.e. it will automatically use java 11 when i am in databricks sdk dir (for example) and java 17 when i am in unity catalog dir (I don't need to go and update the java version everytime i'm working on multiple projects). I imagine a lot of people would be using jenv to manage multiple java versions but i can remove it if it is an issue.